### PR TITLE
install icons to /usr/share and load them from there

### PR DESCRIPTION
**Pull request by hcorion**
This also installs GuiConfigs but we don't load them from /usr/share yet.
This fixes Vulkan freezing at start on linux, and makes the icons show up in the new rsx overlay.
Font loading has yet to be fixed on arch-based distros.

### CODE REVIEW

File: rpcs3/Emu/RSX/overlay_controls.h [hunk 1]
```diff
@@ -457,7 +457,15 @@ namespace rsx
...
 						//Resource was not found in config dir, try and grab from relative path (linux)
 						info = std::make_unique<image_info>(("Icons/ui/" + res).c_str());
-
+#ifndef _WIN32
+						if (info->data == nullptr)
+						{
+							char result[ PATH_MAX ];
+							ssize_t count = readlink( "/proc/self/rpcs3", result, PATH_MAX );
+							auto executablePath = std::string( result, (count > 0) ? count : 0 );
+							info = std::make_unique<image_info>((executablePath + "../share/Icons/ui/" + res).c_str());
```
Comment (myfreeweb) [inline rpcs3/Emu/RSX/overlay_controls.h]: I don't see a `dirname` call… doesn't this result in something like `/usr/local/bin/rpcs3../share/Icons/ui/something`?

File: rpcs3/Emu/RSX/overlay_controls.h [hunk 2]
```diff
@@ -457,7 +457,15 @@ namespace rsx
 					{
 						//Resource was not found in config dir, try and grab from relative path (linux)
 						info = std::make_unique<image_info>(("Icons/ui/" + res).c_str());
-
+#ifndef _WIN32
+						if (info->data == nullptr)
+						{
+							char result[ PATH_MAX ];
+							ssize_t count = readlink( "/proc/self/rpcs3", result, PATH_MAX );
```
Comment (myfreeweb) [inline rpcs3/Emu/RSX/overlay_controls.h]: Is `/proc/self/rpcs3` a thing? o_0 Just looked at `/proc/self` on an Arch VM, the only symlink there is `exe`.

So should be something like:

```cpp
#ifdef __linux__
"/proc/self/exe"
#elif __FreeBSD__
"/proc/curproc/file"
#endif
```

Comment (hcorion) [reply rpcs3/Emu/RSX/overlay_controls.h]: derp, thanks. Will the FreeBSD ifdef be the only BSD one I need?

Comment (myfreeweb) [reply rpcs3/Emu/RSX/overlay_controls.h]: Not sure if any others are running RPCS3, but I guess you can make that just `else`.

**Comment by miguel-nunes** [comment]
Folks, I found a little bug. When installing Atellier Escha & Logy, using the native interface, the instalation starts before finding a controller, breaking the installation giving an error in the dialog -3. If it's needed, I can provide logs of this! Let me know if it's necessary.

**Comment by AniLeo** [comment]
Discuss on #4072 , not here

**Comment by miguel-nunes** [comment]
Sorry @AniLeo , I'll paste this information there

**Pull request merged**